### ISSUE TITLE
Display default language when forms do not have users language code

### ISSUE THREAD

**Issue by newtewt** [medic/medic#6051]
**Is your feature request related to a problem? Please describe.**
When a language is added to the app. If the forms are not updated to include translations for that language we display a bunch of fields to select values however there is no text because there are no translations. 

**Describe the solution you'd like**
We should be able to identify that a form has no translations for the current language. Then display the default language as we do elsewhere in the app. 

**Describe alternatives you've considered**
Another option would be to pump it to the console that the translations are missing. Currently no error is shown. 

**Additional context**

![image](https://user-images.githubusercontent.com/10531066/67322916-bb5f9a80-f4df-11e9-89e5-01607fdf26de.png)

**Comment by SCdF** [comment]
In the core there is [this note](https://github.com/medic/medic/blob/master/webapp/src/js/services/enketo.js#L255-L257):

```js
        // TODO remove this when our enketo-core dependency is updated as the latest
        //      version uses the language passed to the constructor
        currentForm.langs.setAll(options.language);
```

It's somewhat likely that once #4386 is done the solution to this problem changes, so we should probably wait until then.

**Comment by garethbowen** [comment]
I don't think this is possible even with the latest enketo. The way translation works is the HTML is generated with all the translations inline, eg:

```html
<label class="question non-select or-appearance-db-object">
  <span lang="fr" class="question-label" data-itext-id="...">Quel est l'identifiant du patient?</span>
  <span lang="en" class="question-label active" data-itext-id="...">What is the patient's name?</span>
  <input ...>
</label>
````

The labels are all hidden in CSS. When the language is set enketo finds all elements with `[lang="en"]` and [adds the `active` class](https://github.com/enketo/enketo-core/blob/master/src/js/language.js#L88-L93) which shows them. It has no concept of a "default" language to fall back to if the label doesn't contain any spans with the chosen language.

Raise an issue with enketo-core and consider contributing a PR.

**Comment by ngaruko** [comment]
Leaving this screenshot here for testing purposes when the issue is fixed:
![image](https://user-images.githubusercontent.com/6979995/105406858-dd6f3c00-5c91-11eb-9fa4-fa4693d76edb.png)
